docs/hyn/5.4/commands.md

```diff
@@ -49,12 +49,12 @@ Artisan::call('tenancy:run', [
         '--tenant' => [1,2,3]
         ]);
 ```
-> Make sure --tenant value is always an array, an integer value will fail.
+> Make sure --tenant value is always an array. An integer value will fail.
 
 # Recreate command
 
 The recreate command allows you to create databases and filesystem directories for
-all tenants. It will check whether a database already exists, if not it will recreate
+all tenants. It will check whether a database already exists, and if not it will recreate
```

Comment:
```suggestion
all tenants. It will check whether a database already exists. If not it will recreate
```

I think this doesn't make a lot of sense. There's already an `and` further along in the sentence. Let's keep it clearer by making this a new sentence entirely.

Reply:
Agreed. Update committed.